specification/storagemover/resource-manager/custom-words.txt

```diff
@@ -0,0 +1 @@
+NFSauto
```

Comment:
@jianyexi is this OK? I believe we have a custom-words at root level, should we move this word there?

Reply:
yes , please add the custom word in https://github.com/Azure/azure-rest-api-specs/blob/main/custom-words.txt

Reply:
Updated the PR